source/stylesheets/refills/_grid.scss

```diff
@@ -0,0 +1,23 @@
+.grid {
+  $grid-gutter: 0.1rem;
+
+  display: flex;
+  flex-flow: row wrap;
+  margin: 0 - $grid-gutter;
+
+  .grid-box-wrap {
+    flex-basis: 25%;
+    flex-grow: 1;
+    flex-shrink: 0;
```

Comment:
@tysongach I know... On the fence though since I want to be as clear as possible. Allow people who might not be totally familiar with flexbox shorthand to quickly experiment. What do you think about this?